plugins/in_head/in_head.c

```diff
@@ -434,6 +382,52 @@ static int in_head_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */
+static struct flb_config_map config_map[] = {
+    // lines
+    {
+     FLB_CONFIG_MAP_STR, "file", "",
```

Comment:
The default value should be NULL since it errors when pass is not set.
It should be initialization error not be runtime error.

```
d$ bin/fluent-bit -i head -o stdout
Fluent Bit v1.9.0
* Copyright (C) 2015-2021 The Fluent Bit Authors
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[2022/02/23 10:08:50] [ info] [engine] started (pid=38366)
[2022/02/23 10:08:50] [ info] [storage] version=1.1.6, initializing...
[2022/02/23 10:08:50] [ info] [storage] in-memory
[2022/02/23 10:08:50] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
[2022/02/23 10:08:50] [ info] [cmetrics] version=0.2.3
[2022/02/23 10:08:50] [ info] [sp] stream processor started
[2022/02/23 10:08:50] [ info] [output:stdout:stdout.0] worker #0 started
[2022/02/23 10:08:51] [error] [plugins/in_head/in_head.c:93 errno=2] No such file or directory
[2022/02/23 10:08:52] [error] [plugins/in_head/in_head.c:93 errno=2] No such file or directory
[2022/02/23 10:08:53] [error] [plugins/in_head/in_head.c:93 errno=2] No such file or directory
```